### ISSUE TITLE
Miner/mediumquarry

### ISSUE THREAD

**Issue by Howman** [ldtteam/minecolonies#10297]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Are you using the latest MineColonies Version?

- [X] I am running the latest **beta/release** version of MineColonies for my Minecraft version.
I am also running the latest versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [ ] Multi Player

### Minecraft Version

1.21

### MineColonies Version

minecolonies-1.1.719-1.21.1-snapshot

### Structurize Version

structurize-1.0.752-1.21.1-snapshot

### Related Mods and their Versions

neoforge 21.1.51
blockui-1.0.191-1.21.1-snapshot
domum_ornamentum-1.0.204-1.21.1-snapshot
multipiston-1.2.50-1.21.1-snapshot

### Current Behavior

1.quarryer will do nothing unless you drag him to about the center of quarry
2.quarryer keeps saying that the quarry is done and to a sigh him to a new quarry
I am using quarrys from fortress  and original  mineconlonies
If your mine is level 5 before starting quarry is when it says it is done and to start a new quarry
if you go level bye level you have to drag him to center of quarry

### Expected Behavior

well that the miner would start the quarry

### Reproduction Steps

Not sure how to reproduce but i have multiple  miners/quarryers doing this

### Logs

Does  not in logs

### Anything else?

No

### Footer

---
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by LukeAngell38** [comment]
You can also briefly get the quarrier to start on the quarry again if you break and replace the quarry in the same spot.

I am having the same issue in 1.20.1
structurize-1.20.1-1.0.759-snapshot           
minecolonies-1.20.1-1.1.763-snapshot

**Comment by MotionlessTrain** [comment]
I have the feeling that there must be something happening in the colony, causing the quarrier to be caught off guard, and getting confused about the status of their quarry (a raid, rain, nightfall, the quarrier missing items, building being upgraded, etc.). From my own testing, it doesn’t seem to be being out of materials, or things like night or rain, but I have no clue what it can be

**Comment by ZenTrix1** [comment]
Quarry still bugged as of today, a year later.

**Comment by MotionlessTrain** [comment]
This particular issue should already be solved (but got probably missed when it was fixed)
It had to do with unloading the colony

**Comment by MotionlessTrain** [comment]
If you have issues with the quarrier, please open a new issue, and explain what is going wrong, and on which minecolonies version
